src/main/kotlin/org/eclipse/tractusx/bpdmcertificatemanagement/dto/response/CertificateDocumentResponseDto.kt

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.eclipse.tractusx.bpdmcertificatemanagement.dto.response
+
+import io.swagger.v3.oas.annotations.media.Schema
+import org.eclipse.tractusx.bpdmcertificatemanagement.dto.CertificateTypeDto
+import org.eclipse.tractusx.bpdmcertificatemanagement.dto.EnclosedSiteDto
+import org.eclipse.tractusx.bpdmcertificatemanagement.dto.TrustValidatorDto
+import org.eclipse.tractusx.bpdmcertificatemanagement.dto.openapidescription.CommonDescription
+import java.time.Instant
+import java.time.LocalDateTime
+
+data class CertificateDocumentResponseDto(
```

Comment:
I notice there is unique identifier to this certificate. Are you planning to add this later with the query?

Reply:
Yes, I am planning to add documentId as unique identifier.